apps/finance/app/src/components/TransferRow.js

```diff
@@ -116,6 +176,70 @@ class TransferRow extends React.Component {
   }
 }
 
+const Wrap = styled.div`
+  display: inline-flex;
+
+  ${breakpoint(
+    'medium',
+    `
+      display: block;
+    `
+  )};
```

Comment:
Do we need this breakpoint definition if `Wrap` is only used in `renderSmallScreens()`?